### PR TITLE
Added personal stats stats selection

### PR DESCRIPTION
**Pull request by LeoMavri**

### CODE REVIEW

File: lib/User.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { TornAPIBase } from './TornAPIBase';
 import { ITornApiError, IUser, IAmmo, IAttack, IBars, IBasicUser, IBattleStats, ICooldowns, ICrimes, IDiscord, IEducation, IEvents, IGym, IHOF, IIcon, IInventory, IJobPoints, IJobs, IUserCompany, IMedals, IMerits, IMessage, IMoney, INetworth, INotifications, IPerks, IPersonalStats, IRefills, IRevives, IRevivesFull, IUserStock, ITravel, IWorkStats, IUserProperty, IUserSkill, IAttackFull, ILog, IUserStockTransaction, IMissions, IMissionStatus, Errorable } from './Interfaces';
 import axios from 'axios';
+import { TornAPI } from '.';
```
Comment (jgolla) [inline lib/User.ts:4]: This is unused. Can we remove it?

File: test/User.test.ts [hunk 2]
```diff
@@ -58,6 +58,12 @@ describe('User API', () => {
         expect(castedReturn.totalbountyreward).to.equal(245245450245245);
     });
 
+    it('personalstats', async () => {
```
Comment (jgolla) [inline test/User.test.ts:61]: Can we update the name of the test? Maybe some like `personalstats with specified stats`. Also, can we add a spot check on the return of the `personalstats` function to make sure the stats we want come back and more importantly a stat we do not want, does not get returned.

File: test/User.test.ts [hunk 3]
```diff
@@ -58,6 +58,12 @@ describe('User API', () => {
...
+        const stub = sinon.stub(axios, 'get').resolves(TestHelper.getJSON('user_personalstats'));
+        await torn.user.personalstats('123', 123456, ['bazaarsales', 'mailssent', 'rifhits']);
+        expect(stub.args[0][0]).to.equal('https://api.torn.com/user/123?selections=personalstats&stat=bazaarsales,mailssent,rifhits&key=key&timestamp=123456');
+    })
```
Comment (jgolla) [inline test/User.test.ts:65]: Missing a semicolon here. I think.

**Review by jgolla — CHANGES_REQUESTED** [review]

**Comment by LeoMavri** [comment]
All issues have been resolved.

**Review by jgolla — APPROVED** [review]

**Pull request merged**